pulp_smash/tests/platform/api_v2/simple_repo_test.py

```diff
@@ -0,0 +1,174 @@
+# coding=utf-8
+"""Test for basic repo creating functionality.
+"""
```

Comment:
I've reread pep8 a few days ago, and figured out that the recommended way for multiline docstring is like example below ( see https://www.python.org/dev/peps/pep-0008/#documentation-strings for more information):

```python
"""Return a foobang

Optional plotz says to frobnicate the bizbaz first.
"""
```

In other words, the closing `"""` should stay on its own line and don't need a blank line before it.

I'm commenting because I am using the PEP8 recommend format. But as the PEP8 document states, it is a code style guide not a rule.